src/withDaily.ts

```diff
@@ -39,9 +45,14 @@ const withDaily: ConfigPlugin<void> = (
         "android:foregroundServiceType": "camera|microphone"
       },
     })
+
     return config
   })
 
+  if (enableScreenShare) {
```

Comment:
I am adding the screen share support based on these references:
- https://github.com/expo/expo/discussions/18725
-https://github.com/andrew-levy/react-native-safari-extension/tree/main/plugin
-https://github.com/matallui/demo-screen-capture/tree/main/plugin